conda/gateways/repodata/jlap/fetch.py

```diff
@@ -18,6 +18,8 @@
 import zstandard
 from requests import HTTPError
 
+from conda.common.url import mask_anaconda_token
+
```

Comment:
Why the absolute import?

Reply:
We've switched to relative imports a while ago, sounds like we should enforce them with the linter.